sympy/combinatorics/permutations.py

```diff
@@ -1050,6 +1050,12 @@ def _af_new(cls, perm):
         p._size = len(perm)
         return p
 
+    def copy(self):
+        return self.__class__(self.array_form)
+
+    def __getnewargs__(self):
+        return (self.array_form,)
```

Comment:
How about using `_hashable_content`

Reply:
For me the name `_hashable_content` doesn't suggest the entire object can be reconstructed from it.
I see `array_form` used extensively throughout the code to represent this object (and no usage of `_hashable_content`).
WDYS?